app/models/comment.rb

```diff
@@ -108,7 +108,12 @@ def after_update
         users << self.comment_owner
       end
       if notify_user_by_email?(self.comment_owner) && notify_user_of_own_comments?(self.comment_owner)
-        CommentMailer.comment_sent_notification(self).deliver_after_commit
+        if self.reply_comment?
+          parent_comment = self.commentable
+          CommentMailer.comment_reply_sent_notification(parent_comment, self).deliver_after_commit
```

Comment:
You don't have to pass both the comment and its parent as arguments; you can get the parent from the comment inside the mailer action.